tests/php/ORM/PolymorphicHasManyListTest.yml

```diff
@@ -1,20 +1,26 @@
 SilverStripe\ORM\Tests\DataObjectTest\Player:
   multiRelationalPlayer1:
-    FirstName: MultiRelational Player 1
+    FirstName: 'MultiRelational Player 1'
```

Comment:
Applying best practice since I'm already editing this file. Strings should be quoted